airsonic-main/src/main/java/org/airsonic/player/service/SecurityService.java

```diff
@@ -55,11 +66,27 @@
 
     @Autowired
     private UserDao userDao;
+
+    @Autowired
+    private SonosLinkDao sonosLinkDao;
+
     @Autowired
     private SettingsService settingsService;
+
     @Autowired
     private Ehcache userCache;
 
+    @Autowired
+    private Ehcache sonosLinkcodeCache;
+
+    @Autowired
+    private JWTSecurityService jwtSecurityService;
+
+    // This is initialize in GlobalSecurityConfig, is autowired.
```

Comment:
Maybe something like `// This is initialized in GlobalSecurityConfig, which is autowired`.